htdocs/fourn/class/fournisseur.product.class.php

```diff
@@ -1008,7 +1008,7 @@ public function displayPriceProductFournisseurLog($productFournLogList = array()
             //$out .= '<td class="liste_titre right">'.$langs->trans("QtyMin").'</td>';
             $out .= '<td class="liste_titre">'.$langs->trans("User").'</td></tr>';
             foreach ($productFournLogList as $productFournLog) {
-                $out .= '<tr><td class="right">'.dol_print_date($this->db->jdate($productFournLog['datec']), 'dayhour', 'tzuser').'</td>';
+                $out .= '<tr><td class="right">'.dol_print_date($this->db->idate($productFournLog['datec']), 'dayhour', 'tzuser').'</td>';
```

Comment:
idate is a method to use to forge a sql request.
May be we just don't need neither jdate neitheridate here ?